optimum/intel/openvino/utils.py

```diff
@@ -178,6 +181,23 @@ def _is_timm_ov_dir(model_dir):
     return False
 
 
+def _is_open_clip_ov_dir(model_dir, model_name):
+    config_file = None
+    has_xml = False
+    has_bin = False
+    if os.path.isdir(model_dir):
+        for filename in glob(os.path.join(model_dir, "*")):
+            if filename.endswith(f"{model_name}.xml"):
+                has_xml = True
+            if filename.endswith(f"{model_name}.bin"):
+                has_bin = True
+            if filename.endswith("config.json"):
+                config_file = filename
+    if config_file and has_xml and has_bin:
+        return True
+    return False
+
+
```

Comment:
I don't see `_is_open_clip_ov_dir` used anywhere can we remove ? 

```suggestion
```

Reply:
removed